src/ws/cockpitauth.c

```diff
@@ -642,6 +642,13 @@ on_remote_login_done (CockpitSshTransport *transport,
                                            COCKPIT_ERROR_AUTHENTICATION_FAILED,
                                            "Authentication failed");
         }
+      else if (g_str_equal (problem, "no-supported-methods") ||
```

Comment:
The problem name ```no-supported-methods``` is very general. Since problem codes are global, should we:

 1. Just use the already existing ```not-supported```

or 

 2. Create a new code ```authentication-unsupported```

Either one is fine with me.